srcpkgs/lsof/template

```diff
@@ -26,4 +30,8 @@ post_extract() {
 
 post_install() {
 	vlicense COPYING
+
+	# liblsof is still in alpha
+	rm -r "${DESTDIR}/usr/lib"
+	rm -r "${DESTDIR}/usr/include"
```

Comment:
is this necessary with the `--disable-*` flags in configure_args?

Reply:
Yes. Even after adding `--disable-static`, for some reason it still installs `/usr/lib/liblsof.a`

And I don't know anything about `configure` files in order to debug the error.

Reply:
Fixed it by changing Makefile.am